### DISCUSSION TITLE
Updating sliver server/client

### DISCUSSION THREAD

**Discussion by moscowchill** [BishopFox/sliver#1074] [General]
Hi everyone,

I'm not sure as to how I should be updating my installation.
Would someone care 2 explain?

I re-ran installation command and it installed the new client.
However the server is still old version.
The client update command just puts a binary in my home dir and I am sadly not sure what to do with that either...

Thank you.

**Comment by moscowchill** [comment]
It seems that re-running the install command updated the client.
Not sure why it downloaded a server binary however update command now says it's up to date

```
sliver > update

[*] Client v1.5.33 - ce213edab44d33b2c232e0b8dc6c38f7fdabeee7 - linux/amd64
    Compiled at 2023-01-04 18:12:42 +0000 UTC
    Compiled with go version go1.19.4 linux/amd64


[*] Server v1.5.24 - 18ddfd29439311cd5e3b2a1f4a082634ac47637b - linux/amd64
    Compiled at 2022-08-26 18:10:54 +0000 UTC

Checking for updates ... done!

[*] No new releases.
```

**Reply by moloch--** [reply]
This looks like a bug in our version checking. Currently the `update` command will simply download the latest server binary for you but it doesn't install it. To install it you simply copy the new server binary over the old one and restart the process. If you used the Linux install script the server binary will be located at `/root/sliver-server` by default.

**Reply by moscowchill** [reply]
Thank you that fixed it.
Probably a permission thing.
